tests/func/test_warehouse.py

```diff
@@ -1,6 +1,6 @@
 def test_dataset_stats_no_table(cloud_test_catalog, dogs_dataset):
     catalog = cloud_test_catalog.catalog
-    catalog.warehouse.drop_dataset_rows_table(dogs_dataset, 1)
-    num_objects, size = catalog.warehouse.dataset_stats(dogs_dataset, 1)
+    catalog.warehouse.drop_dataset_rows_table(dogs_dataset, version="1.0.0")
+    num_objects, size = catalog.warehouse.dataset_stats(dogs_dataset, version="1.0.0")
```

Comment:
Leftover `int` versions from https://github.com/iterative/datachain/pull/1082; using keyword arguments so they appear the next time someone has to search/replace _en masse_.